cmd/cli/alpha.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+)
+
+const alphaDescription = `
+This command consists of multiple subcommands related that are in alpha.
+`
+
+func newAlphaCmd(out io.Writer) *cobra.Command {
```

Comment:
good question, I am up for either and easy change to make. I modeled it after `kubectl alpha` which I like because it is very clear that when using it.

Reply:
I like this approach, but I am curious to see what other maintainers think.